FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Refactoring function names and comments.
```

Comment:
No changelog necessary, since none of these changes should impact clients

Reply:
Sounds good. Will remove